pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-inject-parameters.adoc

```diff
@@ -1,28 +1,33 @@
-= Inject Parameters
+= Injecting Parameters
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
-The default functions receive the predefined parameters:
+By default, the #[entrypoint] configuration function receives the following injectable parameters:
```

Comment:
This is not accurate. These are not injected by default. They simply can be injected, meaning you could include them as parameters of the entrypoint function.